internal/app/types/friend.go

```diff
@@ -1,7 +1,16 @@
 package types
 
+import (
+	"github.com/globalsign/mgo/bson"
+)
+
 // Friend hold information of a friend
 type Friend struct {
-	ID   string `json:"id" bson:"id"`
-	Name string `json:"name"`
+	Id    bson.ObjectId `bson:"_id" json:"id"`
+	Name        string  `bson:"name" json:"name"`
+	Title       string  `bson:"title" json:"title"`
+	Project     string  `bson:"project" json:"project"`
+	Birthday    string  `bson:"birthday" json:"birthday"`
+	FoodNote    string  `bson:"foodnote" json:"foodnote"`
+	Family      string  `bson:"family" json:"family"`
```

Comment:
How is about other information like: company, office that he/she is working, current position, country, city....?

Reply:
Added